application/actions/service_test.go

```diff
@@ -14,9 +14,15 @@ import (
 	"github.com/gobuffalo/httptest"
 	"github.com/silinternational/wecarry-api/api"
 	"github.com/silinternational/wecarry-api/domain"
+	"github.com/silinternational/wecarry-api/job"
 )
 
 func (as *ActionSuite) Test_serviceHandler() {
+
+	getPost := func(name string) string {
```

Comment:
Maybe a name like `getPostBody` would add a little more clarity. And I guess I'd have put it at package scope (rather than function scope) but that's just a personal preference.

Reply:
I'll make it `postBody`, since I think I remember you don't like to see the prefix `get` on function names and since it's only at function scope. :-)

Reply:
You remember correctly. :-) But it's only a slight preference.